dashboard/bower.json

```diff
@@ -31,7 +31,6 @@
     "jsonlint": "1.6.0",
     "ng-clip": "0.2.6",
     "ng-lodash": "0.2.3",
-    "ng-password-strength": "0.2.1",
```

Comment:
hello Oleksii, could you explain why you need to remove this library to switch style for input boxes ?